test/helpers.bash

```diff
@@ -606,16 +607,16 @@ function check_conmon_cpuset() {
         fi
 
         found_cpuset=$(cat "$cpuset_path/pod_123-456/crio-conmon-$ctr_id/cpuset.cpus")
-        if [ -z "$cpuset" ]; then
+        if [ -z "${cpuset:-}" ]; then
             [[ $(cat "$cpuset_path/pod_123-456/cpuset.cpus") == *"$found_cpuset"* ]]
         else
             [[ "$cpuset" == *"$found_cpuset"* ]]
         fi
     else
         # don't test cpuset if it's not supported by systemd
-        if [[ -n "$systemd_supports_cpuset" ]]; then
+        if [[ -n "${systemd_supports_cpuset:-}" ]]; then
```

Comment:
not needed, we defined systemd_supports_cpuset at the beginning of the function

Reply:
Thanks for spotting, fixed.